src/encode_poc.c

```diff
@@ -0,0 +1,104 @@
+
+#include <x86intrin.h>
+#include <stdio.h>
+#include "shuffle_tables.h"
+
+// POC standalone test until code looks ok
+// usage:  gcc -msse4.1 -o encode_poc encode_poc.c
+
+void dump(__m128i x, char *msg) {
```

Comment:
Testing code should be moved to something like the ``tests`` subdirectory. I.e., we don't want the regular users to get this in their binaries.